### PR TITLE
Update dependency to keycloak version (12.0.4)

### PR DESCRIPTION
**Pull request by badrange**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ https://codesoapbox.dev/keycloak-user-migration
 
 | Keycloak Version | Commit                                                                                                                                           |
 |------------------|--------------------------------------------------------------------------------------------------------------------------------------------------|
-| 11.X             | Current                                                                                                                                          |
+| 12.X             | Current                                                                                                                                          |
```
Comment (daniel-frak) [inline README.md:23]: We should keep 11.X (alongside 12.X being `Current`) as pointing to the last commit with Keycloak 11 dependencies.

**Review by daniel-frak — CHANGES_REQUESTED** [review]

**Comment by daniel-frak** [comment]
Thanks for the contribution!

Quick question, have you checked if everything works okay with this new Keycloak version? I'm asking because of issue #20 - I haven't had time to check it out but that's two people saying they had a problem with it.

File: README.md (hunk 1 above)
Comment (badrange) [reply README.md:23]: Ah, good point! Added a link to the last commit now.

**Comment by badrange** [comment]
> Thanks for the contribution!
> 
> Quick question, have you checked if everything works okay with this new Keycloak version? I'm asking because of issue #20 - I haven't had time to check it out but that's two people saying they had a problem with it.

Thanks for getting back to me! 

I have checked, but not extensively. Our backend for Keycloak is a Drupal 7 site, and the Services module we use doesn't like Keycloak using @'s in the url for creating users :facepalm:  so we ended up having Drupal create users and update passwords using Keycloak's api instead of using this project.

**Pull request merged**